studio/data/sql/execute-sql-query.ts

```diff
@@ -27,7 +38,9 @@ export async function executeSql(
   }
 
   const response = await post(
-    `${API_URL}/pg-meta/${projectRef}/query`,
+    `${API_URL}/pg-meta/${projectRef}/query${
+      queryKey ? `?key=${queryKey.filter((seg) => typeof seg === 'string').join('-')}` : ''
```

Comment:
what is the `?key` param here used for?

Reply:
I was looking at way too many requests to `/query` in the network inspector, so this is purely for debugging in the network inspector haha